### PR TITLE
Fix for issue #71

### PR DESCRIPTION
**Pull request by softwarepk**
Issues with table name identifiers getting generated with invalid characters has been fixed. 
Identifiers will now match the [A-Za-z0-9_] specs.

### CODE REVIEW

**Comment by softwarepk** [comment]
@agronholm for review

File: sqlacodegen/codegen.py [hunk 1]
```diff
@@ -158,8 +158,22 @@ def add_imports(self, collector):
             if len(index.columns) > 1:
                 collector.add_import(index)
 
+    @staticmethod
+    def _convert_to_valid_identifier(name):
```
Comment (agronholm) [inline sqlacodegen/codegen.py:162]: Why did you move the method here? It doesn't seem to be used outside of ModelTable even in your version.

Comment (softwarepk) [reply sqlacodegen/codegen.py:162]: Previously **_convert_to_valid_identifier** was only used in ModelClass and it was a static method of this class. Now **_convert_to_valid_identifier** is also used in ModelTable so it is moved to the base class.

File: sqlacodegen/codegen.py [hunk 2]
```diff
@@ -158,8 +158,22 @@ def add_imports(self, collector):
...
+        elif name == 'metadata':
+            name = 'metadata_'
+
+        return _re_invalid_identifier.sub('_', name)
+
 
 class ModelTable(Model):
+    def __init__(self, table):
+        super(ModelTable, self).__init__(table)
+        self.name = self._convert_to_valid_identifier(table.name)
```
Comment (softwarepk) [inline sqlacodegen/codegen.py:175]: **_convert_to_valid_identifier** used in ModelTable

File: sqlacodegen/codegen.py [hunk 3]
```diff
@@ -208,16 +222,6 @@ def _tablename_to_classname(cls, tablename, inflect_engine):
...
-        assert name, 'Identifier cannot be empty'
-        if name[0].isdigit() or iskeyword(name):
-            name = '_' + name
-        elif name == 'metadata':
-            name = 'metadata_'
-
-        return _re_invalid_identifier.sub('_', name)
-
     def _add_attribute(self, attrname, value):
         attrname = tempname = self._convert_to_valid_identifier(attrname)
```
Comment (softwarepk) [inline sqlacodegen/codegen.py:226]: **_convert_to_valid_identifier** used in ModelClass

File: sqlacodegen/codegen.py (hunk 2 above)
Comment (agronholm) [reply sqlacodegen/codegen.py:175]: Ah, my bad – the diffs can sometimes be confusing and my memory of the code layout is less than perfect :)

**Comment by agronholm** [comment]
Thanks!

**Comment by softwarepk** [comment]
Thanks!
